### PR TITLE
Move 4 IE specific props under HTMLMediaElement

### PR DESCRIPTION
**Pull request by teoli2003**
These properties concern media elements and shouldn't be at the top level. (Part of #16255)

### CODE REVIEW

**Comment by teoli2003** [comment]
Ok, given that merge conflicts can be nasty here, I'll wait for this one to be merged before fixing the conflicts in the author PR around ms* features.

File: files/en-us/web/api/htmlmediaelement/msplaytodisabled/index.md [hunk 1]
```diff
@@ -1,24 +1,21 @@
 ---
-title: msPlayToDisabled
-slug: Web/API/MsPlayToDisabled
+title: HTMLMediaElement.msPlayToDisabled
+slug: Web/API/HTMLMediaElement/msPlayToDisabled
+page-type: api-instance-property
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmediaelement/msplaytodisabled/index.md:4]:
```suggestion
page-type: web-api-instance-property
```

File: files/en-us/web/api/htmlmediaelement/msplaytopreferredsourceuri/index.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 ---
-title: msPlayToPreferredSourceUri
-slug: Web/API/MsPlayToPreferredSourceUri
+title: HTMLMediaElement.msPlayToPreferredSourceUri
+slug: Web/API/HTMLMediaElement/msPlayToPreferredSourceURI
+page-type: api-instance-property
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmediaelement/msplaytopreferredsourceuri/index.md:4]:
```suggestion
page-type: web-api-instance-property
```

File: files/en-us/web/api/htmlmediaelement/msplaytoprimary/index.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
 ---
-title: msPlayToPrimary
-slug: Web/API/MsPlayToPrimary
+title: HTMLMediaElement.msPlayToPrimary
+slug: Web/API/HTMLMediaElement/msPlayToPrimary
+page-type: api-instance-property
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmediaelement/msplaytoprimary/index.md:4]:
```suggestion
page-type: web-api-instance-property
```

File: files/en-us/web/api/htmlmediaelement/msplaytosource/index.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
 ---
-title: msPlayToSource
-slug: Web/API/MsPlayToSource
+title: HTMLMediaElement.msPlayToSource
+slug: Web/API/HTMLMediaElement/msPlayToSource
+page-type: api-instance-property
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmediaelement/msplaytosource/index.md:4]:
```suggestion
page-type: web-api-instance-property
```

File: files/en-us/web/api/htmlmediaelement/msplaytosource/index.md [hunk 5]
```diff
@@ -13,15 +16,9 @@ source associated with the media element for use by the [`PlayToManager`](https:
...
-## Syntax
-
-```js
-ptr = object.msPlayToSource;
-```
-
 ## Value
 
-_PlayTo_ is a means through which an app can connect local playback/display for
+_PlayTo_ is a mean through which an app can connect local playback/display for
```
Comment (wbamberg) [inline files/en-us/web/api/htmlmediaelement/msplaytosource/index.md:21]:
```suggestion
_PlayTo_ enables an app can connect local playback/display for
```

("means to" is probably what was intended but it's perhaps not very clear)

**Review by wbamberg — CHANGES_REQUESTED** [review]

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Pull request merged**